### PR TITLE
Countries

### PR DESCRIPTION
**Pull request by coriander31415**

### CODE REVIEW

File: src/app/shared/components/select-country/select-country.component.html [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <mat-form-field appearance="outline">
-  <mat-select placeholder="{{ 'SETTINGS.SELECT_COUNTRY' | translate}}">
-    <!-- <mat-option *ngFor="let country of countries | keyvalue">
-      <span class="flag-icon flag-icon-{{country.key | lowercase}}"></span> {{ country.value }}
-    </mat-option> -->
+  <mat-label>{{ "SETTINGS.COUNTRY" | translate }}</mat-label>
+  <mat-select placeholder="{{ 'SETTINGS.SELECT_COUNTRY' | translate}}" [(ngModel)]="selectedCountry">
+    <mat-option *ngFor="let country of countriesList" [value]="country.name">{{ country.name }}</mat-option>
```
Comment (kastrubait) [inline src/app/shared/components/select-country/select-country.component.html]: I think it's better to use [value]="country. iso3166CountryCode"

Comment (coriander31415) [reply src/app/shared/components/select-country/select-country.component.html]: Good idea! Done.

**Review by kastrubait — COMMENTED** [review]
image

The text does not fit!

**Comment by coriander31415** [comment]
> image
> 
> The text does not fit!

Fixed

**Pull request merged**